start/sw/web-app/laravel5.md

```diff
@@ -9,41 +9,51 @@ order: 4
 
 Laravel is a high-level PHP Web framework.
 
-#### Install Apache Web Server
-Follow the Apache installation instructions [here](/start/sw/web-app/apache.html).
+#### Install Nginx Web Server
+Follow the nginx installation instructions [here](/start/sw/web-app/nginx.html).
+
 
 #### Install MariaDB Relational Database Server
 Follow the MariaDB installation instructions [here](/tech/database/mariadb/about.html).
 
-#### Install PHP
-Follow the PHP installation instructions [here](/tech/languages/php/php-installation.html).
+Or you can install by : 
```

Comment:
Install [what]? Why would you need to install mysql?

Reply:
You need to setup database when setting up Laravel, MySQL or MariaDB is just the most common one

Reply:
MySQL is just the most popular database used with PHP, and Laravel relies on database to fully function.

Reply:
> You need to setup database when setting up Laravel, MySQL or MariaDB is just the most common one
> MySQL is just the most popular database used with PHP, and Laravel relies on database to fully function.


Well, could you include that information in your guide then?

I'm not trying to argue, nor do I lack the information, I'm merely missing it in the content.

Reply:
Also please omit the `or you can install by`. We try not to duplicate the content.

Reply:
@starvsion, can you deduplicate it? Or shall I?

Reply:
> @starvsion, can you deduplicate it? Or shall I?

you do it ...